website/ocular-config.js

```diff
@@ -79,13 +97,13 @@ module.exports = {
   },
 
   // TODO/ib - remnants from gatsby starter, remove and replace with ocular CAPS constants aboves
-  siteUrl: "https://luma.gl", // Domain of your website without pathPrefix.
-  pathPrefix: "/luma", // Prefixes all links. For cases when deployed to example.github.io/gatsby-advanced-starter/.
-  siteRss: "/rss.xml", // Path to the RSS file.
-  dateFromFormat: "YYYY-MM-DD", // Date format used in the frontmatter.
-  dateFormat: "DD/MM/YYYY", // Date format for display.
-  userName: "WebGL User", // Username to display in the author segment.
-  copyright: "Copyright © 2017 Uber. MIT Licensed", // Copyright string for the footer of the website and RSS feed.
-  themeColor: "#c62828", // Used for setting manifest and progress theme colors.
-  backgroundColor: "#e0e0e0" // Used for setting manifest background color.
+  siteUrl: 'https://luma.gl', // Domain of your website without pathPrefix.
```

Comment:
Can we cut these now? I don't think we use them, they are from the original starter.

Reply:
yes. we should put them somewhere else eventually. let's remove them from ocular-config-template

on a side note - 
idk what to do w/ conflicts here,
i can't rebase cleanly bc the new example glfx is looking for something in @loaders.gl it doesn't get

Reply:
You can comment out the glfx example from the website for now, it is under active development.

Reply:
ok. re the static files (which are the thumbnails file), I'd rather have them in /src, or really anywhere but /static. the reasoning is that they are going to be transformed into much smaller images. only those smaller images will be exposed in the final site. imo, what's in /static is not expected to change and is made available as is